packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -3119,7 +3120,9 @@ export const getAllOfPaginatedService = async <T>(
 
 export type FunctionReturningNextPageToken<T> = (
   nextPageToken?: string | null,
-) => Promise<{ results: T[]; nextPageToken?: string | null }>
+) => Promise<
+  ({ results: T[] } | { page: T[] }) & { nextPageToken?: string | null }
+>
 
 export async function getAllOfNextPageTokenPaginatedService<T>(
```

Comment:
The DownloadListActionsRequired results are in a `page` property instead of a `results` property, so `getAllOfNextPageTokenPaginatedService` needs to be extended to support that